docs/root/version_history/current.rst

```diff
@@ -24,6 +24,7 @@ Minor Behavior Changes
 * quic: add back the support for IETF draft 29 which is guarded via ``envoy.reloadable_features.FLAGS_quic_reloadable_flag_quic_disable_version_draft_29``. It is off by default so Envoy only supports RFCv1 without flipping this runtime guard explicitly. Draft 29 is not recommended for use.
 * router: take elapsed time into account when setting the x-envoy-expected-rq-timeout-ms header for retries, and never send a value that's longer than the request timeout. This behavioral change can be temporarily reverted by setting runtime guard ``envoy.reloadable_features.update_expected_rq_timeout_on_retry`` to false.
 * stream_info: response code details with empty space characters (' ', '\t', '\f', '\v', '\n', '\r') is not accepted by the ``setResponseCodeDetails()`` API.
+* tls: remove TLS 1.0 and 1.1 from server defaults
```

Comment:
```suggestion
* tls: set TLS v1.2 as the default minimal version for servers.
```

And please add a link here to `tls_minimum_protocol_version` (specifying how the default can be modified).

Reply:
In addition, this should be moved to `Incompatible Behavior Changes` for better visibility.

Reply:
OK. Fixed.